src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { action } from '@storybook/addon-actions';
+
+import { SpaceSizes } from '../../theme';
+import { Paragraph } from '../typography';
+import { Stack } from '../layout';
+import Card from './Card';
+import CardHeader from './CardHeader';
+import CardActions from './CardActions';
+import CardContent from './CardContent';
+import CardMedia from './CardMedia';
+import { CardProps } from './Card.types';
+
+const image =
```

Comment:
Should we provide a test for another kind of media like video?

Reply:
yeah, I'm not sure how to do it, I don't want to bring external links in the storybook (as it will lead to flaky tests when the origin is not available) and I also don't want to insert videos into the DS repository.
We probably can do a story with external link and disable the test.

Reply:
Well, there are two ways if you don't want to use urls. The most shitty way is to encode into base64 a low-res, low-quality video just to render something. The another one is how I used to encode videos dynamically in the past. Using canvas. Here is an example: https://stackoverflow.com/questions/66592646/can-i-create-a-video-with-javascript

I think it doesn't worth it. A skipped test will be enough if you want to show an example of the power of DS cards :D

Reply:
Added showcase of video element